src/vs/workbench/contrib/comments/browser/commentThreadWidget.ts

```diff
@@ -401,6 +406,21 @@ export class ReviewZoneWidget extends ZoneWidget implements ICommentThreadWidget
 
 		this._commentsElement = dom.append(this._bodyElement, dom.$('div.comments-container'));
 		this._commentsElement.setAttribute('role', 'presentation');
+		this._commentsElement.tabIndex = 0;
+
+		dom.addDisposableListener(this._commentsElement, dom.EventType.KEY_DOWN, (e) => {
```

Comment:
It might be added to `this._disposables`.

Reply:
@rebornix done, thanks!